test/Parse/availability_query_unavailablility.swift

```diff
@@ -0,0 +1,138 @@
+}
+
+if #unavailable(OSX 10.51) && #unavailable(OSX 10.52) { // expected-error {{expected ',' joining parts of a multi-clause condition}} {{27-30=,}}
+}
+
+
+if #unavailable { // expected-error {{expected availability condition}} expected-error {{closure expression is unused}} expected-error {{top-level statement cannot begin with a closure expression}} expected-note {{did you mean to use a 'do' statement?}} {{17-17=do }}
+}
+
+if #unavailable( { // expected-error {{expected platform name}} expected-error {{expected ')'}} expected-note {{to match this opening '('}}
+}
+
+if #unavailable() { // expected-error {{expected platform name}}
+}
+
+if #unavailable(OSX { // expected-error {{expected version number}} expected-error {{expected ')'}} expected-note {{to match this opening '('}}
+}
+
+if #unavailable(OSX) { // expected-error {{expected version number}}
+}
+
+if #unavailable(OSX 10.51 { // expected-error {{expected ')'}} expected-note {{to match this opening '('}}
+}
+
+if #unavailable(iDishwasherOS 10.51) { // expected-warning {{unrecognized platform name 'iDishwasherOS'}}
+}
+
+if #unavailable(iDishwasherOS 10.51) { // expected-warning {{unrecognized platform name 'iDishwasherOS'}}
+}
+
+if #unavailable(OSX 10.51, OSX 10.52) {  // expected-error {{version for 'macOS' already specified}}
+}
+
+if #unavailable(OSX 10.51, iOS 8.0, *) { }  // expected-error {{platform wildcard '*' is implicit in unavailability}} {{37-38=}}
+
+if #unavailable(iOS 8.0) {
+}
+
+if #unavailable(iOSApplicationExtension, unavailable) { // expected-error 2{{expected version number}}
+}
+	
+// Should this be a valid spelling since `#unvailable(*)` cannot be written?
+if #unavailable() { // expected-error {{expected platform name}}
+}
+
+if #unavailable(OSX 10 { // expected-error {{expected ')' in availability query}} expected-note {{to match this opening '('}}
+}
+
+// Multiple platforms
+if #unavailable(OSX 10.51, iOS 8.0) {
+}
+
+
+if #unavailable(OSX 10.51, { // expected-error {{expected platform name}} // expected-error {{expected ')'}} expected-note {{to match this opening '('}}
+}
+
+if #unavailable(OSX 10.51,) { // expected-error {{expected platform name}}
+}
+
+if #unavailable(OSX 10.51, iOS { // expected-error {{expected version number}} // expected-error {{expected ')'}} expected-note {{to match this opening '('}}
+}
+
+if #unavailable(OSX 10.51, iOS 8.0, iDishwasherOS 10.51) { // expected-warning {{unrecognized platform name 'iDishwasherOS'}}
+}
+
+if #unavailable(iDishwasherOS 10.51, OSX 10.51) { // expected-warning {{unrecognized platform name 'iDishwasherOS'}}
+}
+
+if #unavailable(OSX 10.51 || iOS 8.0) {// expected-error {{'||' cannot be used in an availability condition}}
+}
+
+// Emit Fix-It removing un-needed >=, for the moment.
+
+if #unavailable(OSX >= 10.51) { // expected-error {{version comparison not needed}} {{21-24=}}
+}
+
+// <rdar://problem/20904820> Following a "let" condition with #unavailable is incorrectly rejected
+
```

Comment:
You can remove this line, this has since been fixed. 😃 Maybe also remove the corresponding line from the original file.